site/content/entry/yuzu-progress-report-aug-2020/index.md

```diff
@@ -0,0 +1,209 @@
+
+## Bugfixes and improvements
+
+[Khronos](https://www.khronos.org/) recently added the new `VK_EXT_extended_dynamic_state` extension to the Vulkan API, and driver support is slowly being completed by the vendors. As to be expected with fresh support for this extension, you can expect stuff to be in a *non-perfect* state, or as we like to say, "Expect Dragons". AMD recently added support for this extension in the 20.8.3 Windows driver but part of it is glitched, resulting in graphical errors in games like `Super Mario Odyssey`. [Rodrigo](https://github.com/ReinUsesLisp) temporarily solved this by [blacklisting the extension on the AMD Windows drivers until it is fixed](https://github.com/yuzu-emu/yuzu/pull/4599). Intel doesn't support `VK_EXT_extended_dynamic_state` at the time of writing this, and Nvidia had already fixed their issues with the extension.
+
+{{< single-title-imgs
+    "Mario stared into the abyss, and the abyss stared back (Super Mario Odyssey)"
+    "./smobug.png"
+    "./smofix.png"
+  >}}
+
+Continuing with `VK_EXT_extended_dynamic_state`, a glitch affecting, among others, `Super Smash Bros. Ultimate` was discovered when using this new extension. Attack sprites were missing during gameplay. [Rodrigo](https://github.com/ReinUsesLisp) [solved the issue on all compatible drivers.](https://github.com/yuzu-emu/yuzu/pull/4555).
+
+{{< single-title-imgs
+    "The Master Sword can't just look like a regular sword (Super Smash Bros. Ultimate)"
+    "./smashbug.mp4"
+    "./smashfix.mp4"
+  >}}
+
+Thumbnail pictures in in-game save slots were a black rectangle for most games up until now. By [fixing block linear copies](https://github.com/yuzu-emu/yuzu/pull/4453), [Rodrigo](https://github.com/ReinUsesLisp) added better immersion by avoiding loading up a wrong save or course by mistake in games such as `The Legend of Zelda: Breath of the Wild` and `Super Mario Maker 2`.
+
+{{< single-title-imgs
+    "Ohh, this one was my good course! (Super Mario Maker 2)"
+    "./smm2bug.png"
+    "./smm2fix.png"
+  >}}
+  
+## Precision has a cost
+
+Up until now, CPUs lacking the FMA instruction set had to run generic code as fallback. After a lot of work, [Merry](https://github.com/MerryMage) added [optimizations intended for CPUs lacking the `FMA instruction set`](https://github.com/yuzu-emu/yuzu/pull/4541). 
+
+What this means is that users with CPUs from the AMD Phenom II series or older, Intel 3rd generation Core-i series or older, and all Pentium, Celeron and Atom CPUs can now select the "Unsafe" setting and see major performance improvements in almost all games. Don't bother to try this setting on CPUs that have FMA as it will not improve your performance in that case.
+
+Due to being a less precise alternative, expect Dragons.
```

Comment:
Change to `However, due to being a less precise alternative, expect emulation inaccuracies if you decide to enable it. Here be dragons, as they say.`